File: web-manager/src/main/webapp/js/service/orderListService.js
app.service('orderListService',function ($http) {
    //查询所有并展示
    this.findAll=function () {
       return $http.get('../orderList/findAll.do');
    }
    //分页查询
  this.findPage=function (page, rows) {
        return $http.get('../orderList/findPage.do?page='+page+'&rows='+rows);
    }
    // //订单
    this.getOrderList=function (sellerId) {
        return $http.get('../orderList/getOrderList.do?sellerId='+sellerId);
    }
})

File: interface/src/main/java/cn/itcast/core/service/OrderListService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.order.Order;
import cn.itcast.core.pojo.order.OrderItem;
import cn.itcast.core.pojo.seller.Seller;

import java.util.List;

public interface OrderListService {
    //查询订单所有
    List<Seller> findAll();
    //分页查询
    PageResult findPage(Integer page, Integer rows);
//    //多条件分页查询
//    PageResult search(Integer page, Integer rows, Order order);
    //根据商品ID查询所有
    List<OrderItem> getOrderList(String sellerId);
}
